src/render/quantum.rs

```diff
@@ -462,20 +496,20 @@ impl AudioRenderQuantum {
     }
 
     /// Convert to a single channel buffer, dropping excess channels
-    pub fn force_mono(&mut self) {
+    pub(crate) fn force_mono(&mut self) {
```

Comment:
I was thinking we could even drop this one, but as it is `pub(crate)` probably less important now